root/etc/e-smith/events/actions/nethserver-dc-user-modify

```diff
@@ -40,3 +49,29 @@ if ($? != 0) {
     die("[ERROR] User $userName modification failed\n");
 }
 
+
+#Create a ldif file and use it to modify the shell access
+#a random name and a chmod is done for security
+my $filename = "/tmp/$userName-". sprintf("%08X", rand(0xFFFFFFFF)).".ldif";
+my $contentFile = qq (
+dn: cn=$userName,cn=Users,$baseDN
+changetype: modify
+replace: loginShell
+loginShell: $shell
+);
+
+open(my $fh, '>', $filename) or die "Could not open file '$filename' $!";
+chmod 0600, $filename;
+print $fh $contentFile;
+close $fh;
+
+system('/usr/bin/ldapmodify', 
```

Comment:
I'd go with ``ldbmodify`` (similar to ldapmodify, but running on bare .ldb files) in the nsdc container. To test:

    systemd-run -M nsdc -t /bin/bash
    ldbmodify ...

You could run it directly with systemd-run, or wrapping in an ``expect`` script, like we're (sadly) doing in [other actions](https://github.com/NethServer/nethserver-dc/blob/master/root/etc/e-smith/events/actions/nethserver-dc-user-create#L51).

Reply:
ok I will have a go on this clue